packages/wrangler/src/r2/helpers.ts

```diff
@@ -438,23 +436,22 @@ export async function tableFromNotificationGetResponse(
 		event_type: string;
 	}[]
 > {
-	const reducer = async ([_, { queue, rules }]: [
-		DetailID,
-		NotificationDetail[DetailID],
-	]) => {
-		const queueResp = await queueIdentifier(config, queue);
+	const reducer = async (entry: GetQueueDetail) => {
 		const rows = [];
-		for (const { prefix = "", suffix = "", actions } of rules) {
+		for (const {
+			prefix = "",
+			suffix = "",
+			actions,
+			ruleId,
+			createdAt = "",
+		} of entry.rules) {
 			rows.push({
-				queue_name: queueResp.queue_name,
+				rule_id: ruleId,
+				created_at: createdAt,
+				queue_name: entry.queueName,
 				prefix,
 				suffix,
-				event_type: Array.from(
-					actions.reduce((acc, action) => {
-						acc.add(eventCategoryByAction[action]);
-						return acc;
-					}, new Set<R2EventType>())
-				).join(","),
+				event_type: actions.join(","),
```

Comment:
formatLablelledValues handled indentation so think could be concatenated with a new line
```suggestion
				event_type: actions.join("\n"),
```